main.tf

```
@@ -0,0 +1,8 @@
+resource "aws_resourcegroups_group" "test" {
```

We use `this` for any resource that we expect to only have a single reference of in a module.

```suggestion
resource "aws_resourcegroups_group" "this" {
```